spikeinterface/widgets/figurl/timeseries.py

```diff
@@ -17,34 +17,17 @@ class TimeseriesPlotter(FigurlPlotter):
     def do_plot(self, data_plot, **backend_kwargs):
         assert HAVE_TILED_IMAGE, ("To use the figurl backend for timeseries, you neeed the 'figurl_tiled_image'. "
                                   "Install it with >>> pip install figurl_tiled_image")
-        if "clim" in data_plot:
-            clim = data_plot["clim"]
-            if np.array(clim).ndim == 1:
-                clims = [clim] * len(data_plot["layer_keys"])
-            else:
-                assert len(clim) == len(data_plot["layer_keys"])
-                clims = clim
-        else:
-            clims = [-200, 200] * len(data_plot["layer_keys"])
-
-        if "tile_size" in data_plot:
-            tile_size = data_plot["tile_size"]
-        else:
-            tile_size = 512
-            
-        if "num_timepoints_per_row" in data_plot:
-            num_timepoints_per_row = data_plot["num_timepoints_per_row"]
-        else:
-            num_timepoints_per_row = 6000
+        tile_size = data_plot.get("tile_size", 512)
+        num_timepoints_per_row = data_plot.get("num_timepoints_per_row", 6000)
```

Comment:
Why do we need default value security here ?
For me all defaults have to be handle in the base.
All plotters can digest without checks the data_plot dict.